views/scss/qti/_all-interactions.scss

```diff
@@ -7,6 +7,9 @@
             }
         }
     }
+    .interaction-cover {
+        display:none;
+    }
```

Comment:
again, if it is creator specific, it is better to have only creator scss modified